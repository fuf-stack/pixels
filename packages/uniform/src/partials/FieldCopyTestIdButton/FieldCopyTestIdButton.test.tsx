import { it } from 'vitest';

it.todo('not working right now');

// import { describe } from 'vitest';

// import storySnapshots from '@repo/storybook-config/story-snapshots';

// import * as stories from './FieldCopyTestIdButton.stories';

// describe('Story Snapshots', () => {
//   storySnapshots(stories);
// });
