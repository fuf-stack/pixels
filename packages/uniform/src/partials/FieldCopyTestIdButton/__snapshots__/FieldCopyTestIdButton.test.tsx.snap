// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Story Snapshots > Default 1`] = `
<div>
  <button
    class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap font-normal subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-tiny gap-2 rounded-small px-0 !gap-0 transition-transform-colors-opacity motion-reduce:transition-none text-default-foreground data-[hover=true]:bg-default/40 min-w-8 w-8 h-8 bg-transparent pointer-events-auto"
    tabindex="0"
    type="button"
  >
    <span
      data-testid="icon-FaBullseye"
    >
      FaBullseye
    </span>
  </button>
</div>
`;
